execution_workflows/APA-Scan/workflow/Snakefile

```diff
@@ -0,0 +1,357 @@
+"""
+Snakemake pipeline for APA-Scan.
+For help see: https://snakemake.readthedocs.io/en/stable/index.html.
+"""
+
+
+
+
+
+##### READ THIS ###############################################################
```

Comment:
Okay, guess I should have read that first. Not sure how to deal with that though. Are you absolutely sure you didn't change the way that APA-Scan works? Because otherwise we'd be benchmarking _you_ rather than the published tool and that might put us in a line of fire. In any case, the better approach would be to fork their repository, add the required changes, then build a Docker image that contains your fork rather than the original. I would also open a PR to the original repo from that fork, letting the authors know about your changes, why you did them and what they would fix. This would at least give them the chance to respond to them, so that we can hopefully - eventually - run the workflow with a version that is sanctioned by the authors

Reply:
Yeah I've tried multiple times, the only think I could think is to use the exact directories they specify - but I'm not able to do hthat on my work PC because I cant make new dirs in root. Cheers for the advice.